doc/md/deployment/intro.md

```diff
@@ -29,8 +29,9 @@ configuration data.
 For now, you can choose any of the supported databases:
 1. SQLite (file) (in-memory is also supported but does not require encryption)
 2. MySQL
-3. MariaDB
-4. PostgresDB
+3. TiDB
```

Comment:
did we test TiDB as a backing DB for management UI storage? 
i know it _should_ work, but..

Reply:
just tried TiDB as a backing DB for management UI storage on my mac, it works perfectly 🎉

Reply:
@hooopo Thank you! :)